### PR TITLE
US: maintenance

### LINKED ISSUE #3335: Fox Business
**Channel Name:** FOX Business Chnnel
**Country:** USA
**Language:** English
**External Link (optional):**
https://www.foxbusiness.com/
**Notes (optional):** xxx

### LINKED ISSUE #3457: [Fix/Change needed] US Channels
Channels have been reported to not work properly by @jkemp814.

That's where we see that it really needs a fix and clean x)

Pull Requests that are giving fixes : #3458 (by @Dum4G)

Broken Streams : 
- [x] MTV Hits Europe (fixed on it's own)
- [ ] <s> ESPN U</s>
- [x] Docurama
- [x] Decentric Media (fixed on it's own)
- [x] Dancestar TV (twice)
- [ ] <s> Crime Story (STIRR Version)</s>
- [x] Cooking Panda (Source 2)  (fixed on it's own)
- [x] CONtv Anime (XUMO Version)
- [x] CONtv
- [x] Complex
- [ ] <s>Combat Go</s>
- [x] Circle
- [x] Cinemax
- [x] Cine Romantico (XUMO Version)
- [x] Cheddar (twice)
- [x] WUSA-HD (CBS 9 Washington DC)
- [x] CBC News
- [x] BSpokeTV
- [x] Boston 25 News (WFXT-TV)
- [x] Boomerang (twice)
- [x] Bon Appetit
- [x] Bloomberg TV US (4 times)
- [x] Bloomberg TV US Normal + Politics Live Event
- [x] Blaze Live (Pluto TV shut off their LLNWD server)
- [x] Black News Channel (twice) (XUMO Bad Request - replacement link : https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxblacknewschannel/CDN/playlist.m3u8)
- [ ] <s> Beverly Hillbillies (STIRR Version) (Google DAI - Server answers, but 404 Not Found on links)</s>
- [x] Beast Mode (Amazon AWS telling bad session ID, I think we got a few AWS links that match this pattern, to be fixed)
- [x] Battery Pop (XUMO going weird, replacement link : https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxbatterypop/CDN/playlist.m3u8)
- [ ] <s> Bambu (Server answers with invalid playlist)</s>
- [x] Baeble (XUMO going weird, replacement link : https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxbaeble/CDN/playlist.m3u8)
- [x] BabyFirst (access denied, replacement link is https://livecdn.fptplay.net/qnetlive/babyfirst_hls.smil/playlist.m3u8)
- [x] 30A TV
- [ ] <s>21 Jump Street (STIRR) (Google DAI - Server answers, but 404 Not Found on links)</s>
- [x] ABC 9 Orlando FL (Maybe @MapGuy11 can host this channel onto his proxy)
- [x] AXN (Indonesia) (Fails to connect)
- [x] AXN (0 Bytes playlist)
- [x] Automoto (Bumblebee) (Replacement link (found by @Sphinxroot : https://stitcheraws.unreel.me/wse-node01.powr.com/live/5bf220fad5eeee0f5a40941a/playlist.m3u8)
- [x] Architectural Digest (XUMO going weird, replacement link : https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxarchitecturaldigest/CDN/playlist.m3u8)
- [ ] <s>AMC Presents (Plex/Wurl version) (Server answers with playlist but 403 Forbidden when playing it)</s>
- [x] America's Test Kitchen (XUMO) (XUMO weird again, replacement link : https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxamericastestkitchen/CDN/playlist.m3u8)
- [x] AMC All Weddings (by WEtv) (Plex/Wurl version) (Server answers with playlist but 403 Forbidden when playing it)
- [ ] <s>All Time (XUMO) (0 byte playlist)</s>
- [x] AFV Spanish (404 Not Found)
- [x] Adult Swim (Replacement link given, and present on Dum4G PR)
- [x] AMC Absolute Reality (by WEtv) (Plex/Wurl version) (Replacement link given : https://amc-absolutereality-1.samsung.wurl.com/manifest/playlist.m3u8 (I think it's already in playlist))

Names to be changed

- [x] Key West - Florida Keys (Proposed change : Beach TV (Key West - Florida Keys))
- [x] Myrtle Beach - Grand Strand (Proposed change : Beach TV (Myrtle Beach - Grand Strand)
- [x] Panama City (Proposed change : Beach TV (Panama City)
- [x] Florida - Alabama (Proposed change : Beach TV (Florida - Alabama))

### LINKED ISSUE #3469: Add: NBC 5 (KSDK)
**Channel Name:** NBC 5 (KSDK)
**Country:** USA
**Language:** English
**External Link (optional):** https://livevideo01.ksdk.com/hls/live/2014965/newscasts/live.m3u8
**Notes (optional):** St. Louis, Missouri

Website: https://www.ksdk.com/
Stream URL: https://www.ksdk.com/watch

### LINKED ISSUE #3470: Add: Nine PBS
**Channel Name:** Nine PBS
**Country:** USA
**Language:** English
**External Link (optional):** https://ketcdt.lls.cdn.pbs.org/ketcdt/0436e066-85c7-408b-a4fd-5b492ccb5fd8/primary.m3u8
**Notes (optional):** St. Louis, Missouri [Geo-blocked]

### LINKED ISSUE #3473: Add: All Sports
**Channel Name:** All Sports Television Network
**Country:** US
**Language:** English
**External Link (optional):** https://2-fss-2.streamhoster.com/pl_120/amlst:203932-1476320/playlist.m3u8
**Notes (optional):** Digital over-the-air sports channel. Source is from KGPT-CD: https://kgpttv26.com/stream-all-sports

### PR DESCRIPTION
**Pull request by Dum4G**
Removes what's dead
Reinstates what needs to be reinstated
New sources
Old requests
Minor improvemements

Closes https://github.com/iptv-org/iptv/issues/3457
Closes https://github.com/iptv-org/iptv/issues/3470
Closes https://github.com/iptv-org/iptv/issues/3473
Closes https://github.com/iptv-org/iptv/issues/3469
Closes https://github.com/iptv-org/iptv/issues/3335

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
All good for me, you may also correct that PBS Georgia mistake that was inside MapGuy's PR.

File: channels/us.m3u [hunk 1]
```diff
@@ -6586,7 +6588,7 @@ https://wxeldt.lls.cdn.pbs.org/wxeldt/6f553836-7e24-445a-8c3c-cc418aaba9dc/prima
 https://wfsuhd.lls.cdn.pbs.org/wfsuhd/da7db8ae-5c3b-450e-9a25-76aa661eac79/primary.m3u8
 #EXTINF:-1 tvg-id="WEDU.us" tvg-name="WEDU-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wedu-color-cobranded-logo-87oN0he.png" group-title="General",PBS Tampa (WEDU) [Geo-blocked]
 https://wedudt.lls.cdn.pbs.org/wedudt/c4ab80c1-ca8c-4a6e-a8b5-c5995919792b/primary.m3u8
-#EXTINF:-1 tvg-id="WVAN.us" tvg-name="WVAN-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wgtv-color-single-brand-logo-IeTkPR9.png" group-title="General",PBS [Geo-blocked]rgia (WVAN) (WXGA) [Geo-blocked]
+#EXTINF:-1 tvg-id="WVAN.us" tvg-name="WVAN-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wgtv-color-single-brand-logo-IeTkPR9.png" group-title="General",PBS Georgia (WVAN) (WXGA) [Geo-blocked]
```
Comment (LaneSh4d0w) [inline channels/us.m3u:6591]: WVAN-TV Savannah, Georgia

WXGA isn't in this stream.

Comment (Dum4G) [reply channels/us.m3u:6591]: Both stations are 135km away from each other. Broadcasted content is the same I guess

**Review by MapGuy11 — APPROVED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:6591]: Sounds fair to me.

**Review by BellezaEmporium — APPROVED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (MapGuy11) [reply channels/us.m3u:6591]: Yes PBS Georgia is a statewide network instead of the affiliate in Atlanta so I took their two top stations and put it in.

**Pull request merged**